src/observer.js

```diff
@@ -273,6 +248,32 @@ const reactiveMixin = {
     }
 }
 
+function makeObservableProp(target, propName) {
+    const valueHolderKey = Symbol(propName + " value holder")
+    const atomHolderKey = Symbol(propName + " atom holder")
```

Comment:
I am wondering whether using Symbols is a breaking change. So far we didn't depend on them, but I think React itself already requires Symbol or a polyfill of it? (Since mobx-react 5 can still be used with mobx 4 so should work on any ES 5 env)

Reply:
I will check this out, maybe there is a way to do this without symbols.

Reply:
The support looks pretty good to me: https://caniuse.com/#search=symbol

Reply:
without symbols it could be done by putting an object under some ugly key. Symbols is preferred though :) You could always create a super naive polyfill like: `function Symbol(name) { return `$mobxReactProp${symbolname}${Math.random()} }`. But since react itself already requires quite some polyfills ([link](https://reactjs.org/docs/javascript-environment-requirements.html]) I think we can bet on Symbol to be present. And otherwise fix it if somebody reports is